.editorconfig

```diff
@@ -5,10 +5,14 @@ root = true
 charset = utf-8
 end_of_line = lf
 indent_size = 2
-indent_style = space
+indent_style = tab
```

Comment:
Can we keep this as spaces?

Reply:
The justfile requires tab otherwise the recipes are considered invalid. I can just add a separate section for the justfile and then change the default back to space

Reply:
Done here https://github.com/ethereum-optimism/superchain-registry/pull/439/commits/05657800bd6f2964e71971aed6a350727880a12d